doc/purescript-vim.txt

```diff
@@ -68,6 +70,15 @@ INDENTATION                *purescript-Indentation*
 To configure indentation in `purescript-vim` you can use the following variables:
 
 
+G:PURESCRIPT_DISABLE_INDENT                *purescript-g-purescript_disable_indent*
+
+Disable indentation altogether.
+
+>
+    let g:purescript_indent_case = 1
```

Comment:
I know this is generated documentation, but wanted to point out the same thing here.